src/main/python/rlbot/csv/items.py

```diff
@@ -0,0 +1,7 @@
+import os
+from pathlib import Path
+
+
+def open_csv():
+    """Opens RLBot's items.csv file for reading"""
+    return open(os.path.join(Path(os.path.realpath(__file__)).parent, "items.csv"), "r")
```

Comment:
Many people consider it a best practice to use a `with` statement when reading from a file: https://stackoverflow.com/questions/14776853/is-it-good-practice-to-depend-on-pythons-with-as-statement

Are you intending for people to do something like this?
```python
with open_csv() as csv:
   ...
```

That's not very obvious to me, might be good to add a type annotation and a comment, or just return a Path object instead. A Path might be more broadly useful.

Reply:
Sure, I'll make the change.

Also, yes that was how I hoped they would use it: https://github.com/RLBot/RLBotPack/blob/10d0cc969a91087c8d874548b86f3ead1d979c9b/RLBotPack/Kamael_family/nebulous_loadout.py#L11